include/rppt_tensor_logical_operations.h

```diff
@@ -86,6 +86,29 @@ RppStatus rppt_bitwise_and_host(RppPtr_t srcPtr1, RppPtr_t srcPtr2, RpptDescPtr
 RppStatus rppt_bitwise_and_gpu(RppPtr_t srcPtr1, RppPtr_t srcPtr2, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+#ifdef GPU_SUPPORT
+/*! \brief Exclusive OR computation on HIP backend for a NCHW/NHWC layout tensor
+ * \details This function computes exclusive OR of corresponding pixels for a batch of RGB(3 channel) / greyscale(1 channel) images with an NHWC/NCHW tensor layout.<br>
+ *          srcPtr depth ranges - Rpp8u (0 to 255), Rpp16f (0 to 1), Rpp32f (0 to 1), Rpp8s (-128 to 127).
+ *          dstPtr depth ranges - Will be same depth as srcPtr.
+ * \image html img150x150.png Sample Input1
+ * \image html img150x150_2.png Sample Input2
+ * \image html logical_operations_exclusive_or_img150x150.png Sample Output
```

Comment:
please upload this output image in the following folder
rpp/docs/data/doxygenOutputs/
https://github.com/ROCm/rpp/blob/b9c767827589d38c327a964987edfecac554fee2/docs/data/doxygenOutputs/logical_operations_bitwise_or_img150x150.png

Reply:
Updated